jina/peapods/networking.py

```diff
@@ -451,6 +451,48 @@ async def task_wrapper(requests, stubs):
 
         return asyncio.create_task(task_wrapper(requests, connection))
 
+    @staticmethod
+    def get_grpc_channel(
+        address: str,
+        options: Optional[list] = None,
+        asyncio: Optional[bool] = False,
+        https: Optional[bool] = False,
+        root_certificates: Optional[str] = None,
+    ) -> grpc.Channel:
+        """
+        Creates a grpc channel to the given address
+
+        :param address: The address to connect to, format is <host>:<port>
+        :param options: A list of options to pass to the grpc channel
+        :param asyncio: If True, use the asyncio implementation of the grpc channel
+        :param https: If True, use https for the grpc channel
+        :param root_certificates: The path to the root certificates for https, only used if https is True
+
+        :return: A grpc channel or an asyncio channel
+        """
+
+        secure_channel = grpc.secure_channel
+        insecure_channel = grpc.insecure_channel
+
+        if asyncio:
+            secure_channel = grpc.aio.secure_channel
+            insecure_channel = grpc.aio.insecure_channel
+
+        if options is None:
+            options = GrpcConnectionPool.get_default_grpc_options()
+
+        if https:
+            credentials = grpc.ssl_channel_credentials()
+
+            if root_certificates is not None:
```

Comment:
I think its okay to pass None as root_certificates to ssl_channel_credentials(), see https://grpc.github.io/grpc/python/_modules/grpc.html#ssl_channel_credentials

so I would simplify line 485 to 488 to the single line
credentials = grpc.ssl_channel_credentials(
                    root_certificates=root_certificates
                )
no first call without args and no if needed?